Dockerfile-dc

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.10
+
+RUN apk add -U --no-cache ca-certificates
+
+FROM golang:latest
+
+ADD . /app
+WORKDIR /app
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
+RUN CGO_ENABLED=0 GO111MODULE=on GOOS=linux GOARCH=amd64 packr2
```

Comment:
`packr2` does not need these arguments - see the makefile for reference.

Reply:
Will do!

Reply:
We would still need the GO111MODULE=on argument, this ensures that packr2 doesn't build the boxes as per the path instead of the modules.

Reply:
right, that makes sense. You can set `ENV GO111MODULE on` in the dockerfile so go modules are enabled for all commands

Reply:
Yup, we could do that too!